### DISCUSSION TITLE
cURL error 35: error:14077438:SSL routines:SSL23_GET_SERVER_HELLO:tlsv1 alert internal error

### DISCUSSION THREAD

**Discussion by CryptosWORLD** [postalserver/postal#1866] [Help with using Postal]
hi, i receive this error in a Mailwizz  after add **Postal** as mail server with **API**
and send **test a mail to Validate this server and the config** : 

17#page-info-validate)cURL error 35: error:14077438:SSL routines:SSL23_GET_SERVER_HELLO:tlsv1 alert internal error

cURL error 35: [error:14077438:SSL](url) routines:SSL23_GET_SERVER_HELLO:tlsv1 alert internal error

any solutions ?

![ScreenShot_20220215011406](https://user-images.githubusercontent.com/39472963/153963102-e06b1b08-36de-4270-bc4a-5421389c79f5.png)

ps:
AlmaLinux release 8.5 (Arctic Sphynx) (full updated)
docker 20.10.12
postal 2.1.0
Mailwizz 2.1.3

thank you

**Comment by willpower232** [comment]
have you enabled tls in the SMTP config?

**Reply by CryptosWORLD** [reply]
from postal side i didnt see something like that 
from mailwizz there are 6 ways i have try them all 3+ times each :
1 - 2) Postal web api  (http or https) (error : Here is a transcript of the error message: Couldn’t send message to API  )

3) Postal smtp  (error : Failed to authenticate on SMTP server with username "USERNAME@DOMAIN.com" using the following authenticators: "CRAM-MD5", "LOGIN", "PLAIN". Authenticator "CRAM-MD5" returned "Symfony\Component\Mailer\Exception\TransportException: Expected response code "235" but got code "535", with message "535 Denied". )
4) **Postal smtp with tls** (error:  Connection could not be established with host "ssl://mail.DOMAIN.com:25": stream_socket_client(): SSL operation failed with code 1. OpenSSL Error messages:
error:140770FC:SSL routines:SSL23_GET_SERVER_HELLO:unknown protocol )
5) Postal smtp with ssl  (error:  Connection could not be established with host "ssl://mail.DOMAIN.com:25": stream_socket_client(): SSL operation failed with code 1. OpenSSL Error messages:
error:140770FC:SSL routines:SSL23_GET_SERVER_HELLO:unknown protocol)
6) Postal smtp with starttls   (error:  Failed to authenticate on SMTP server with username "USERNAME@DOMAIN.com" using the following authenticators: "CRAM-MD5", "LOGIN", "PLAIN". Authenticator "CRAM-MD5" returned "Symfony\Component\Mailer\Exception\TransportException: Expected response code "235" but got code "535", with message "535 Denied

i have tested multiple times in port 25 and 2525...

**Reply by willpower232** [reply]
> Failed to authenticate on SMTP server with username "USERNAME@DOMAIN.com"

Have you made a credential within Postal? the username should be `something\something else`
